File: ZZGG/DataModels/AccountChampionStats.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DataModel
{
    public class AccountChampionStats
    {
        public int ChampionId { get; set; }
        public int ChampionLevel { get; set; }
        public Int64 ChampionPoints { get; set; }
        public Int64 LastPlayTime { get; set; }
        public Int64 ChampionPointsSinceLastLevel { get; set; }
        public int ChampionPointsUntilNextLevel { get; set; }
        public bool ChestGranted { get; set; }
        public int TokensEarned { get; set; }
        public string SummonerId { get; set; }

    }
}

File: ZZGGAPP/src/app/models/accountChampionStats.ts
export class AccountChampionStats{
    public championId: number | undefined;
    public championLevel : number | undefined;
    public championPoints : number | undefined;
    public lastPlayTime : number | undefined;
    public championPointsSinceLastLevel : number | undefined;
    public championPointsUntilNextLevel : number | undefined;
    public chestGranted : boolean | undefined;
    public tokensEarned  : number | undefined;
    public summonerId : string | null = "";
    public championName : string | null = "";
    public championIcon : string | null = "";
    public championTitle : string | null = "";
}
